runtime-display/glfw/spec

```diff
@@ -1,5 +1,4 @@
-VER=3.3.2
-REL=1
+VER=3.3.8
 SRCS="tbl::https://github.com/glfw/glfw/releases/download/$VER/glfw-$VER.zip"
```

Comment:
Please try not to use automatically packaged tarballs, use Git tags instead.

Reply:
Fixed.